Sources/Bond/Observable Collections/TreeChangeset+Array2D.swift

```diff
@@ -126,6 +126,20 @@ extension MutableChangesetContainerProtocol where Changeset: TreeChangesetProtoc
     public func removeAllItemsAndSections() {
         removeAll()
     }
+
+    /// Replace items of a section at the given index with new items.
+    public func replaceItems(ofSectionAt sectionIndex: Int, with newItems: [Item]) {
+        self[sectionAt: sectionIndex].items = newItems
```

Comment:
This will send `update` event with the section index so calling `descriptiveUpdate` afterwards seems redundant.